sdk-java/src/main/java/io/littlehorse/sdk/wfsdk/UserTaskOutput.java

```diff
@@ -6,4 +6,8 @@ public interface UserTaskOutput extends NodeOutput {
     public UserTaskOutput withNotes(WfRunVariable notes);
 
     public UserTaskOutput withNotes(LHFormatString notes);
+
+    UserTaskOutput withOnCancelException(String exceptionName);
```

Comment:
Nit pick: `withOnCancellationException`. That's slightly more semantic, even though it's longer to type.